common/src/main/java/org/conscrypt/NativeCrypto.java

```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
+/*
```

Comment:
This is the pattern that other projects that I've seen work (including Netty) when borrowing code snippets. It shouldn't have any impact on users because Apache 2.0 is compatible.  In fact, we already have other files that are dual licensed (again with Netty).